src/Azs.Network.Admin/test/Get-AzsLoadBalancer.Tests.ps1

```diff
@@ -1,3 +1,19 @@
+$envFile = 'env.json'
+Write-Host "Loading env.json"
+if ($TestMode -eq 'live') {
+    $envFile = 'localEnv.json'
+}
+if (Test-Path -Path (Join-Path $PSScriptRoot $envFile)) {
+    $envFilePath = Join-Path $PSScriptRoot $envFile
+} else {
+    $envFilePath = Join-Path $PSScriptRoot '..\$envFile'
+}
+$env = @{}
+if (Test-Path -Path $envFilePath) {
+    $env = Get-Content (Join-Path $PSScriptRoot $envFile) | ConvertFrom-Json
+    $PSDefaultParameterValues = @{"*:SubscriptionId" = $env.SubscriptionId; "*:Tenant" = $env.Tenant; "*:Location" = $env.Location; "*:ResourceGroupName" = $env.ResourceGroup }
+}
+
```

Comment:
Remove this section from all tests and add default parameters for location and resource group name in utils.ps1 setEnv function.

Reply:
Removed and created a loadEnvJson.ps1 to be imported.